### DISCUSSION TITLE
Moving compare translations function into tool panel

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#553] [Ideas]
How about moving the compare translations function into the newly designed tool panel?

In that case the idea would be to have a permanent view on other translations - based on the currently selected verse(s). This means there would be a third panel button (aside from tag list and dictionary) - for comparing translations. In case of the compare translations function, the content of the tool panel would automatically refresh when you change the verse selection and it would only show other translations than the currently active one.

**Comment by tobias-klein** [comment]
@debritto @augustin-colesnic @zhuiks
Any thoughts on this one?

**Reply by debritto** [reply]
Awesome! Brilliant idea.

**Comment by zhuiks** [comment]
Yes! 
Initially, my idea was to have another section/window at the bottom of the screen for the various details for the selected verse, including  compare translations. But giving it more thought, I'm leaning towards using the same tool window that we have now.

This also can be done for cross-references.

**Reply by debritto** [reply]
Perfect! In this way you maintain simplicity and acessibility. I can't wait to see it in action.

**Comment by tobias-klein** [comment]
I created an issue: #581
Have a look at the draft PR for this feature - including a demo: #582

**Reply by debritto** [reply]
I had take a look at the demo and I find it very good. Perhaps the window could even bring a icon to copy the verses content.

**Reply by tobias-klein** [reply]
Thanks for the feedback! Yes - the icon/button for clipboard copy would make sense in the context of #537, which you raised recently. I would suggest to handle this separately as a future improvement.

**Reply by debritto** [reply]
Awesome!
